### ISSUE TITLE
Enhancement request: Ability to specify  log message time when using service/logger/insertMessage API call

### ISSUE THREAD

**Issue by achuchev** [Axway/ats-httpdblogger#2]
Currently, when you insert new message (using service/logger/insertMessage API) the current time is used as log message time. This cause a problem if the log message has different time. We want to have a way to specify custom log message time. 

Example: 
```
POST /ats-httpdblogger/service/logger/insertMessage
{
	"machineName" : "10.240.102.51",
	"sessionId" : "5F5D36F393DC81142A6A7FE2665600F6",
	"threadName" : "MainThread",
	"message" : "My Log Message",
	"level" : "INFO",
	“timestamp”: 1493709734
}
```

The timestamp is new optional parameter which specifies the log message time. When not specified, the current time will be used (as today).

**Comment by ngogovski** [comment]
It does make sense to support an optional timestamp parameter for all events.
Do you agree?

**Comment by achuchev** [comment]
Yes, I agree. Any ETA for this?

**Comment by dslaveykov** [comment]
Hello, request has been implemented.

**Comment by achuchev** [comment]
Exellent. Thanks for the quick turnaround.

**Comment by dslaveykov** [comment]
Closing issue. Feature is in current snapshot and will be available in version 4.0.1
